### PR TITLE
navitime.co.jp

### LINKED ISSUE #109543: navitime.co.jp
### Issue URL (Ads)
 [https://www.navitime.co.jp/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.navitime.co.jp%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard/AdGuard Pro for iOS v4.3.2
Browser: | Safari
DNS filtering: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Other:</b><br/>AdGuard Experimental<br/><br/><b>Language-specific:</b><br/>AdGuard Japanese, <br/>AdGuard Chinese

### PR DESCRIPTION
**Pull request by SKEIDs**
Fix #109543

### CODE REVIEW

File: AnnoyancesFilter/sections/mobile-app.txt [hunk 1]
```diff
@@ -22,6 +22,10 @@
 ##head[prefix~="medium-com:"] ~ body .js-openInAppButton
 ##head[prefix~="medium-com:"] ~ body .js-postActionsBarContent > .js-readNextInteractions
 !
+navitime.co.jp###smartapp-banner-area
+navitime.co.jp###footer-app-dl-frame
+navitime.co.jp##.footer-invite-app-section
+navitime.co.jp##.app-download-footer-frame
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/mobile-app.txt:28]:
```
navitime.co.jp##.footer-invite-app-section
navitime.co.jp##.app-download-footer-frame
```
Not reproduced.

File: JapaneseFilter/sections/specific.txt [hunk 2]
```diff
@@ -2188,6 +2188,8 @@ navitime.co.jp###under-advertisement
 navitime.co.jp##.section-ad-1
 navitime.co.jp##.section-ad-3
 navitime.co.jp##.section-ad-4
+navitime.co.jp##.footer-rect-ad-frame
+navitime.co.jp###main-page-frame > div[style="display:flex;justify-content:center;align-items:center;height:120px;width:100%;"]
```
Comment (Alex-302) [inline JapaneseFilter/sections/specific.txt:2192]: `navitime.co.jp###main-page-frame > div[style="display:flex;justify-content:center;align-items:center;height:120px;width:100%;"]`

Not reproduced.

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/mobile-app.txt (hunk 1 above)
Comment (SKEIDs) [reply AnnoyancesFilter/sections/mobile-app.txt:28]: These elements are here. UA is mobile. 
URL: https://www.navitime.co.jp/transfer/pass/?from=top.topics
<details><summary>navitime.co.jp##.footer-invite-app-section</summary>


</details><br/>

<details><summary>navitime.co.jp##.app-download-footer-frame</summary>


</details><br/>

File: JapaneseFilter/sections/specific.txt (hunk 2 above)
Comment (SKEIDs) [reply JapaneseFilter/sections/specific.txt:2192]: URL: https://www.navitime.co.jp/transfer/pass/?from=top.topics

<details><summary>navitime.co.jp###main-page-frame > div[style="display:flex;justify-content:center;align-items:center;height:120px;width:100%;"]</summary>


</details><br/>

**Review by Alex-302 — APPROVED** [review]
Ok, thanks for links.

**Pull request merged**